drag_and_drop_v2/default_data.py

```diff
@@ -19,7 +19,9 @@
 MIDDLE_ZONE_DESCRIPTION = _("Use this zone to associate an item with the middle layer of the triangle.")
 BOTTOM_ZONE_DESCRIPTION = _("Use this zone to associate an item with the bottom layer of the triangle.")
 
-ITEM_CORRECT_FEEDBACK = _("Correct! This one belongs to {zone}.")
+ITEM_CORRECT_FEEDBACK_TOP = _("Correct! This one belongs to The Top Zone.")
+ITEM_CORRECT_FEEDBACK_MIDDLE = _("Correct! This one belongs to The Middle Zone.")
+ITEM_CORRECT_FEEDBACK_BOTTOM = _("Correct! This one belongs to The Bottom Zone.")
 ITEM_INCORRECT_FEEDBACK = _("No, this item does not belong here. Try again.")
 ITEM_NO_ZONE_FEEDBACK = _("You silly, there are no zones for this one.")
```

Comment:
So this is the file that contains the default values of this XBlock strings @pomegranited mentioned in this comment [comment](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/172#discussion_r212787389) in PR #172.
Answering the question, it looks weird actually to have see variables surrounded by a translation took, and it wasn't obvious that we're translating default variables here. I liked the idea of having default values and strings, but we might improve this in another PR maybe and implement a better approach for having these values translated.

Reply:
👍